integration-tests/tests/src/index.ts

```diff
@@ -27,8 +27,8 @@ afterEach(() => {
   }
 });
 
-import "./utils/import-app.test";
-import "./utils/chai-plugin.test";
+import "./utils/import-app.test.ts";
+import "./utils/chai-plugin.test.ts";
```

Comment:
Is there a way to detect these extensions automatically?

Reply:
Yeah, I played around with it. It think it gets confused about the `.test` part of the extension 😕

Reply:
I was getting this issue as well when I was getting the react-native tests to run again, but in a different section.  Wasn't really able to resolve it.  Very odd.